modules/portmaster/src/app/widgets/notification-widget/notification-widget.scss

```diff
@@ -12,6 +12,7 @@
   @apply bg-background;
 
   &:not(.empty) {
+    // TODO(ppacher): Remove min-height, always adjust height to notification content if one is open.
```

Comment:
@ppacher

Reply:
Yeah it wasn't that easy and worked so far so I didn't fix it. Do you need it or can you ignore it?

Reply:
I'll add it to the backlog.